proposal/COMP110 architecture essay/comp110 essay proposal.tex

```diff
@@ -0,0 +1,72 @@
+\item[web link:] \url{http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6575955&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6575955}
+\item[Full text link:]
+\item[Comments:]this paper introduces a new technique which looks for the closest distance between two objects. this can make colision detection more effective.
+\end{description}
+
+\section*{Paper 4}
+
+\begin{description}
+\item[Title:] Interactive Mesostructures withVolumetric Collisions
+\item[Citation:]\cite{<nykl2014interactive>}
+\item[Abstarct:]"This paper presents a technique for interactively colliding with and deforming mesostructures at a per-texel level. It is compatible with a broad range of existing mesostructure rendering techniques including both safe and unsafe ray-height field intersection algorithms. This technique is able to replace traditional 3D geometrical deformations (vertex-based) with 2D image space operations (pixel-based) that are parallelized on a GPU without CPU-GPU data shuffling and integrates well with existing physics engines. Additionally, surface and material properties may be specified at a per-texel level enabling a mesostructure to possess varying attributes intrinsic to its surface and collision behavior. Furthermore, this approach may replace traditional decals with image-based operations that naturally accumulate deformations without inserting any new geometry. This technique provides a simple and efficient way to make almost every surface in a virtual world responsive to user actions and events. It requires no preprocessing time and storage requirements of one additional texture or less. The algorithm uses existing inverse displacement map algorithms as well as existing physics engines and can be easily incorporated into new or existing game pipelines."
+\item[web link:]\url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6799298}
+\item[Full text link:]
+\item[Comments:]This paper looks into a new technique for interactively coliding with and deforming mesostructures at a par text level. it is a new algorithm which can be used with existing physics engines and new games so it can be a more effective physics engine for games.it allows for accumulating deformations without the need for new geomatry. 
+\end{description}
+
+\section*{Paper 5}
+
+\begin{description}
+\item[Title:] Collision course by transformation of coordinates and plane decomposition
+\item[Citation:]\cite{<bendjilali2009collision>}
```

Comment:
Looks interesting -- unfortunately, I do not have access to Cambridge University Press.